webpack.config.cjs

```diff
@@ -197,6 +197,40 @@ const dashboard = {
   },
 };
 
+const webStoriesScripts = {
+  ...sharedConfig,
+  entry: {
+    'web-stories-scripts': './includes/assets/js/index.js',
+    'carousel-script': './includes/assets/js/carousel-view.js',
```

Comment:
Can they live in `assets/src` instead?

```
'carousel': './assets/src/carousel/index.js',
'lightbox': './assets/src/lightbox/index.js',
```

Reply:
certainly, can do that. I had kept these scripts here thinking that since scripts under `assets/src` are mainly used in `admin` and styles under `includes/assets` are mainly for the FE.